learning-development/learning-support.qmd

```diff
@@ -48,23 +53,76 @@ In this workshop, we cover:
 
 Contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk) to register interest in workshops happening at your site or to request new topics!
 
-## Partnership programme
+---
 
- *	Dedicated resource to support development work, either building skills or working on analytical or statistical outputs.
- *	A short-term project with an agreed set of goals and timescale decided as part of an initiation meeting.
- * Contact statistics.development@education.gov.uk if you're interested in finding out more about the support we can give.
+## Partnership programmes
 
-## Specific resources
+The Statistics Development Team offers bespoke partnership programmes for analyst teams across the department, designed as dedicated resource to support development work, either building skills or filling resource gaps to allow improvements in analytical or statistical outputs.
 
-We also have specific learning resources and materials for [SQL](../learning-development/sql.html), [R](../learning-development/r.html), and [Git](../learning-development/git.html)
+Partnerships are designed as short-term projects, agreed in conjunction with the team and their Grade 6. The maximum length for the majority of partnership programmes will be three months, and the project must be clearly defined at the outset.
+
+The aim of partnership programmes is always to upskill the team throughout the project. For this reason, we require any teams taking part to commit time to upskilling during the partnership and being willing to independently continue the work after the partnership is complete, if appropriate. 
+
+---
+
+### What could a partnership programme involve?
+
+---
+
+- Flexible skilled resource for your team to support with R, Git, SQL or Databricks
+- Help with dedicated upskilling 
+- Knowledge sharing, including around best practice and accessibility, and peer review
+- Support with RAP implementation to allow your team to meet the RAP baseline or best practice criteria
+
```

Comment:
Is there one about just pointing to best practice / helping to decide on and keep work the right track

Reply:
Is that what our day to day job is - so we should encourage that outside of partnership programmes rather than in here, like we don't need a team to have a partnership to just understand they're on the right track with implementing RAP, that should be BAU-style work?

Reply:
I was trying to acknowledge difference between tier 1 and tier 2 but maybe didn't use the right words. I agree where we can do this via BAU style work we should and that's enough (over creating partnerships that don't need to be partnerships).

Reply:
I think that's kind of what I was trying to get at with this bit:

> Knowledge sharing, including around best practice and accessibility, and peer review

Have changed to:

> Knowledge sharing, including support to make decisions on best practice and accessibility, and peer review

Does that work?

Reply:
Works for me! Resolving